### PR TITLE
check added for parent course not approved state

### PR DESCRIPTION
**Pull request by tasawernawaz**
[ECOM-6290](https://openedx.atlassian.net/browse/ECOM-6290)

- [x]  [ECOM-7158](https://openedx.atlassian.net/browse/ECOM-7158) front end design.
- [x]  [ECOM-7157](https://openedx.atlassian.net/browse/ECOM-7157) add "approved parent course" check to all states.

**Screenshot:**

screen shot 2017-02-17 at 2 46 23 pm

### CODE REVIEW

**Comment by tasawernawaz** [comment]
@awais786 @waheedahmed Please review.

File: course_discovery/apps/publisher/tests/test_views.py [hunk 1]
```diff
@@ -1103,6 +1103,22 @@ def _data_for_review_button(self):
...
         self.course_run.staff.add(PersonFactory())
 
+    def test_parent_course_not_approved(self):
+        """ Verify that if parent course is not approved than their will be a message
+        shown on course run detail page that user can't submit for approval.
+        """
+        self.user.groups.add(Group.objects.get(name=INTERNAL_USER_GROUP_NAME))
+
+        response = self.client.get(self.page_url)
+        self.assertNotIn(response.content.decode('utf-8'), '<div class="parent-course-approval">')
```
Comment (waheedahmed) [inline course_discovery/apps/publisher/tests/test_views.py]: `self.assertNotContains(response, '<div class="parent-course-approval">')`

File: course_discovery/templates/publisher/_approval_widget.html [hunk 2]
```diff
@@ -4,6 +4,20 @@
     <div class="margin-top20">
         <h5 class="hd-5 emphasized approvals-heading">{% trans "APPROVALS" %}</h5>
     </div>
+
+    {% if is_course_run %}
+      {% if not object.course.course_state.is_approved %}
```
Comment (waheedahmed) [inline course_discovery/templates/publisher/_approval_widget.html]: You can achieve above both in single statement.
`{% if is_course_run and not object.course.course_state.is_approved %}`

**Review by waheedahmed — CHANGES_REQUESTED** [review]

**Comment by tasawernawaz** [comment]
@waheedahmed Made suggested changes, Please take another look.

**Review by waheedahmed — APPROVED** [review]

**Review by awais786 — APPROVED** [review]

**Pull request merged**